docs/themerc

```diff
@@ -7,12 +7,13 @@
 
 # general
 border.width: 1
-padding.width: 0
-padding.height: 3
 
-# The following options has no default, but fallbacks back to
-# font-height + 2x padding.height if not set.
-# titlebar.height:
+#
+# We do not support the global padding.{width,height} of openbox because
+# the default labwc button geometry has deviates from that of openbox
```

Comment:
Nit: "has deviates" reads awkwardly. Maybe "the default button geometry of labwc deviates"?

Reply:
Thanks. Will fix this latet.